layouts/shortcodes/keps-data.html

```diff
@@ -0,0 +1,42 @@
+<script src="/js/keps/kep_search.js"></script>
+<div >
+  <dir class="d-flex">
+    <div class="col">
+      <div class="input-group mt-2 col-md-12">
+        <input type="text" name="search" id="keps_search" class="form-control" placeholder="Search KEP(s) by keyword" />
+      </div>
+    </div>
+    <div class="col ml-auto">
+      <div class="input-group mx-auto mt-2 col-md-12" style="width: 400px;">
+        <label for="keps">Choose SIG group:</label>
+        <select class="custom-select" name="sig group" id="sig_group_select">
+          <option selected="selected" value="all">Choose...</option>
+          {{ range $data_sig := .Site.Data.sigs.sigs.sigs }}
+          <option value="{{$data_sig.dir}}">{{ printf "%s" $data_sig.name }}</option>
+          {{ end }}
+        </select>
+      </div>
+    </div>
+  </dir>
+  <table id="keps_table" class="table table-bordered table-hover mt-2 col-md-12">
+    <thead>
+      <tr>
+        <th scope="col">KEP Number</th>
+        <th scope="col">KEP Title</th>
+        <th scope="col">SIG-Group</th>
+        <th scope="col">Author</th>
+      </tr>
+    </thead>
+    <tbody>
```

Comment:
The sort order right now is weird. I think it's unsorted. Ideally, it should be ordered by the KEP number in ascending order.

Reply:
:+1: Ordering by KEP number seems good for an initial release.

Reply:
Good point. It is better to have option to sort keps while generating the YAML data using `kepctl query` API. For now I will try it using jquery.

Reply:
You can also use https://gohugo.io/functions/sort/

eg
```go
{{ range $index, $element := sort $.Site.Data.keps.keps_data.keps "kep_number" }}
```

(bear in mind that `kep_number` is a string and that this might not sort how you hoped)

Reply:
Thanks!